db/featured.ts

```diff
@@ -14,17 +14,6 @@ export const FEATURED_PROJECTS = [
     twitterURL: '',
     documentationURL: '',
   },
-  {
-    color: '#0280BE',
-    logo: 'rancher-desktop.svg',
-    tag: 'development',
-    name: 'Rancher Desktop',
-    description: 'Kubernetes on your desktop.',
-    href: 'https://rancherdesktop.io/',
-    repositoryURL: 'https://github.com/rancher-sandbox/rd',
-    twitterURL: '',
-    documentationURL: '',
-  },
```

Comment:
Rancher Desktop is still being actively developed and shipped. I don't think it needs to be removed.

Reply:
I talked with Mike back then IIRC, the point was that Rancher Desktop was scaled down but not dead, yet would unlikely receive lots of development - hence why removing it from the featured projects rotation but not the full list of projects.
If we want to maintain the impression externally, I'm fine with that as well.